package-builder/build-manifest.nix

```diff
@@ -196,7 +196,7 @@ pkgs.runCommandNoCC name
                 # and we're not using env -i like we are in the impure build
 
                 FLOX_SRC_DIR=$(pwd) FLOX_RUNTIME_DIR="$TMP" \
-                  ${flox-env-package}/activate --env ${flox-env-package} --env-project $(pwd) --mode build-sandbox -- \
+                  ${flox-env-package}/activate --env ${flox-env-package} --mode build --env-project $(pwd) -- \
```

Comment:
note nonblocking: this is still pretty different than the activation in flox-build.mk - is there a reason not to make the env handling consistent? Seems like it will be easier to keep behavior consistent if the invocation is consistent

Reply:
You're right - I forgot to add `envFilter` into the invocation, and it turns out that doing so fixes the one problem that I was seeing - sending an [ugly] patch now to see if that fixes things, and will clean it up tomorrow.

Reply:
commit 752711413604fa9607e9930b7c7406228830dd24 cleans this up - resolving conversation